src/main/java/List.java

```diff
@@ -0,0 +1,61 @@
+public class List {
+    private static int amountOfItems = 0;
+    private static Task[] tasks;
+
+
+    public List() {
+        tasks = new Task[100];
```

Comment:
The use of magic number here is ambiguous, feel free to consider using a constant